### PR TITLE
Added cards

### PR DESCRIPTION
**Pull request by nishasen**
made 4 four types of card, yet to make the horizontal card

### CODE REVIEW

File: Stylesheet/Components/card.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+.card-overlay {
+    display: grid;
+    grid-template-columns: auto;
+    grid-template-rows: auto auto;
+}
+
+.card-overlay .card-title{
+    padding-left: 1rem;
+    padding-right: 1rem;
```
Comment (MaruthiPothuganti) [inline Stylesheet/Components/card.css]: Suggestion: can use shorthand property

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended to use Shorthand property

File: Stylesheet/Components/card.css (hunk 1 above)
Comment (nishasen) [reply Stylesheet/Components/card.css]: Okay changing that.

File: playground.html [hunk 2]
```diff
@@ -12,21 +12,23 @@
...
     
+
+    
     
     
-     
-      
-    <!-- <script src="/js/alert.js" type="text/javascript"></script> -->
+    
+    <!-- <script src="/js/alert.js" type="text/javascript"></script> -->
```
Comment (tanishq20) [inline playground.html:24]: If you are not using these scripts then avoid giving like this comment.

**Review by tanishq20 — COMMENTED** [review]
One small clarification.  
Good to merge.

File: playground.html (hunk 2 above)
Comment (nishasen) [reply playground.html:24]: That folder is just for testing and keeping things, I will remove it later

**Comment by nishasen** [comment]
[Playground - Google Chrome 2022-02-02 17-59-13.zip](https://github.com/nishasen/Tungstar/files/7986402/Playground.-.Google.Chrome.2022-02-02.17-59-13.zip)

File: Stylesheet/css/general.css [hunk 3]
```diff
@@ -3,15 +3,7 @@ html, body {
...
-    max-width: 100%;
-    height: auto;
-}
-
-.img-round {
-    border-radius: 50%;
+    font-family: var(--st-subheader-text);
 }
 
 .img-square {
```
Comment (kedark152) [inline Stylesheet/css/general.css:9]: Please check this class .img-square syntax

**Review by kedark152 — COMMENTED** [review]
Please check the class syntax for general.css. otherwise everything looks good.

File: Stylesheet/css/general.css (hunk 3 above)
Comment (nishasen) [reply Stylesheet/css/general.css:9]: Yes checked, done it, thank you

**Pull request merged**